### ISSUE TITLE
PYI - Pyrinhash Algo missing

### ISSUE THREAD

**Issue by jeromemorignot** [RainbowMiner/RainbowMiner#2613]
Hi,

Would you be able to add the Pyrinhash algo? Rigel and lolminer can mine it. Herominers and other pools are available.

Thanks!

**Comment by RainbowMiner** [comment]
It is already online and working. It goes by the name HeavyhashPyrin within Rainbowminer. Just add `"PYI": "pytin:yourpyrinhashwalletaddress",` to Herominers in pools.config.txt and off you go.

**Comment by papadrams** [comment]
@RainbowMiner You should also add it to be autocreated and autobenchmarked when mrr pool is selected.
Also also, the calculations on pyi on both herominers and kryptex are way off. Its appears x2 more profitable than it actually is

**Comment by sfalas01** [comment]
@RainbowMiner btw Kryptex does not support mining with email (for now) for Pyrin and Karlsen. They only support direct mining for now.

**Comment by RainbowMiner** [comment]
> You should also add it to be autocreated and autobenchmarked when mrr pool is selected.

Done :) It should arrive at your systems within the hour.

**Comment by jeromemorignot** [comment]
> It is already online and working. It goes by the name HeavyhashPyrin within Rainbowminer. Just add `"PYI": "yourpyrinhashwalletaddress",` to Herominers in pools.config.txt and off you go.
Yep - that worked, should have been more thourough! Thanks for the quick response.

**Comment by RainbowMiner** [comment]
> Also also, the calculations on pyi on both herominers and kryptex are way off. Its appears x2 more profitable than it actually is

Solved with the last release v4.9.1.3

**Comment by RainbowMiner** [comment]
Pyrinhash has been added - it is in Data/algorithms.json and supported by lolminer, MiniZ, Teamred and SrbMinerMulti268. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
